python/python/bystro/proteomics/tests/test_fragpipe_utils.py

```diff
@@ -0,0 +1,25 @@
+import re
+import pytest
+
+import pandas as pd
+
+from bystro.proteomics.fragpipe_utils import check_df_cols
+
+
+def test_check_df_cols_happy_path():
```

Comment:
Could you add docstrings here and throughout?

Reply:
So while I agree in general about docstrings, I'd be specifically wary of making them mandatory in tests, because the primary problem with testing is getting people to write them at all, and so things that increase the friction of writing tests can backfire.  

Happy to add them in this PR but I think it's worth getting general consensus on about whether to require them.